src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -729,3 +731,18 @@ def push_to_argilla(
             UserWarning,
         )
         return self
+
+    # TODO(@frascuchon): is this actually needed? what are the probabilities on a missmatch happening?
+    def push_to_huggingface(self, repo_id: str, generate_card: Optional[bool] = True, *args, **kwargs) -> None:
+        """Pushes the current `FeedbackDataset` to HuggingFace Hub.
+
+        Note:
+            The records from the `RemoteFeedbackDataset` are being pulled before pushing,
+            to ensure that there's no missmatch while uploading those as those are lazily fetched.
+
+        Args:
+            repo_id: the ID of the HuggingFace repo to push the dataset to.
+            generate_card: whether to generate a dataset card or not. Defaults to `True`.
+        """
```

Comment:
I think we can notify user about what's going on with remote datasets and push_to_huggingface method ("the dataset will be pulled locally and pushed from that, if you want to push it several times, pull it first using `ds.pull()` method)

cc @davidberenstein1957